internal/sirius/add_feedback.go

```diff
@@ -0,0 +1,63 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/ministryofjustice/opg-sirius-user-management/internal/model"
+	"net/http"
+)
+
+func (c *Client) AddFeedback(ctx Context, form model.FeedbackForm) error {
+	var body bytes.Buffer
+	var err error
+
+	if len(form.Message) == 0 {
+		return ValidationError{
+			Message: "isEmpty",
+		}
+	}
+
+	if len(form.Message) > 900 {
```

Comment:
Maybe have this len(form.Message) as a var and that means you only have to look at it once?

Reply:
I'm not sure I follow, if it was a variable I'd only use this once?